src/sftp.tf

```diff
@@ -52,24 +52,6 @@ resource "azurerm_private_endpoint" "sftp_blob" {
   tags = var.tags
 }
 
-resource "azurerm_eventgrid_system_topic" "sftp" {
```

Comment:
@matteocellucci This topic and the related subscription are still needed, since events are generated (and can be captured on eventgrid/eventhub) when files are deposited on storage account via a standard rest interface, which is the interface we use to deposit files for our client